.github/workflows/main.yaml

```diff
@@ -91,6 +91,16 @@ jobs:
         use-mamba: true
         miniforge-variant: Mambaforge
 
+    - uses: nanasess/setup-chromedriver@v1
+      - run: |
+          export DISPLAY=:99
+          chromedriver --url-base=/wd/hub &
+          sudo Xvfb -ac :99 -screen 0 1280x1024x24 > /dev/null 2>&1 & # optional
+
+    - name: run epigraphhub tests
+      - run: |
```

Comment:
here "run" is not a new step .. is just part of the "run epigraphhub test" step
so you don´t need a "-" here